r2r/vector_dbs/lancedb/base.py

```diff
@@ -39,18 +41,49 @@ def initialize_collection(
             self, collection_name: str, dimension: int
     ) -> None:
         self.collection_name = collection_name
+
         try:
-            import pyarrow  # TODO ADD Dependency
+            import pyarrow
         except ImportError:
             raise ValueError(
                 f"Error, `pyarrow` is not installed. Please install it using `pip install pyarrow`."
             )
```

Comment:
Hey. Pyarrow is dependency for lancedb. You probably don't need to check if it installed, only lancedb should be checked

Reply:
Not sure if that's the case: https://github.com/lancedb/lancedb/blob/main/python/pyproject.toml

Reply:
Lancedb depends on lance with had pyarrow as a dependency. Im from lancedb team :)

Reply:
Done!

Reply:
Can you double check please? I still see the pyarrow dependency check